src/main/java/blusunrize/immersiveengineering/common/data/Recipes.java

```diff
@@ -41,14 +46,20 @@
 import java.util.Map.Entry;
 import java.util.function.Consumer;
 
-class Recipes extends RecipeProvider
+public class Recipes extends RecipeProvider
 {
 	private final Path ADV_ROOT;
 	private final HashMap<String, Integer> PATH_COUNT = new HashMap<>();
 
 	private final int standardSmeltingTime = 200;
 	private final int blastDivider = 2;
 
+	public static final DeferredRegister<IRecipeSerializer<?>> RECIPE_SERIALIZERS = new DeferredRegister<>(ForgeRegistries.RECIPE_SERIALIZERS, ImmersiveEngineering.MODID);
+
+	public static final RegistryObject<IRecipeSerializer<?>> SPEEDLOADER_LOAD = RECIPE_SERIALIZERS.register("crafting_special_speedloader_load", () -> new SpecialRecipeSerializer<>(SpeedloaderRecipe::new));
+	public static final RegistryObject<IRecipeSerializer<?>> POTION_BULLET_FILL = RECIPE_SERIALIZERS.register("crafting_special_potion_bullet_fill", () -> new SpecialRecipeSerializer<>(PotionBulletRecipe::new));
+	public static final RegistryObject<IRecipeSerializer<?>> FLARE_BULLET_COLOR = RECIPE_SERIALIZERS.register("crafting_special_flare_bullet_color", () -> new SpecialRecipeSerializer<>(FlareBulletRecipe::new));
```

Comment:
Please move this somewhere else, I don't want the `data`-package to do anything in non-datagenerator runs. Similar to how the `client`-package shouldn't be touched on a dedicated server. Also, why are these `IRecipeSerializer<?>`'s rather than a more specific generic? Is that simply a limitation of `DeferredRegister`?

Reply:
didn't chose a more specific generic, because I didn't think it mattered, because the only thing potentially regularly calling it is IRecipeSerializer getSerializer(). changed that.

as for moving the stuff: a good idea where to?
I have to keep these 3 lines together with `RECIPE_SERIALIZERS`, otherwise I can't simply keep them static final and have to figure out how to properly initialize them across different classes.